blocks/regex_command_event.js

```diff
@@ -0,0 +1,283 @@
+            "types": ["number", "date", "unspecified"]
+        }
+    ],
+
+    options: [
+        {
+            "id": "command_restriction",
+            "name": "Command Restriction",
+            "description": "Description: The user will not be able to use this command if a command restriction applies.",
+            "type": "SELECT",
+            "options": {
+                "none": "None",
+                "server_only": "Server Only",
+                "server_owner_only": "Server Owner Only",
+                "bot_owner_only": "Bot Owner Only",
+                "dms_only": "DMs Only"
+            }
+        },
+        {
+            "id": "required_member_permission",
+            "name": "Required Member Permission",
+            "description": "Description: The required member permission to use this command.",
+            "type": "SELECT",
+            "options": {
+                "none": "None",
+                "administrator": "Administrator",
+                "create_instant_invite": "Create Instant Invite",
+                "kick_members": "Kick Members",
+                "ban_members": "Ban Members",
+                "manage_channels": "Manage Channels",
+                "manage_guild": "Manage Server",
+                "add_reactions": "Add Reactions",
+                "view_audit_log": "View Audit Log",
+                "priority_speaker": "Priority Speaker",
+                "stream": "Video",
+                "view_channel": "View Text/Voice Channel(s)",
+                "send_messages": "Send Messages",
+                "send_tts_messages": "Send TTS Messages",
+                "manage_messages": "Manage Messages",
+                "embed_links": "Embed Links",
+                "attach_files": "Attach Files",
+                "read_message_history": "Read Message History",
+                "mention_everyone": "Mention Everyone",
+                "use_external_emojis": "Use External Emojis",
+                "view_guild_insights": "View Server Insights",
+                "connect": "Connect (Connect to a voice channel)",
+                "speak": "Speak (Speak in a voice channel)",
+                "mute_members": "Mute Members (Mute members across all voice channels)",
+                "deafen_members": "Deafen Members (Deafen members across all voice channels)",
+                "move_members": "Move Members (Move members between voice channels)",
+                "use_vad": "Use Voice Activity",
+                "change_nickname": "Change Nickname",
+                "manage_nicknames": "Manage Nicknames (Change other members' nicknames)",
+                "manage_roles": "Manage Roles",
+                "manage_webhooks": "Manage Webhooks",
+                "manage_emojis": "Manage Emojis"
+            }
+        },
+        {
+            "id": "case_sensitive",
```

Comment:
Case sensitivity can be controlled via the regex expression, this option can be removed.

Reply:
As already said, you can remove this option entirely.